episodes/04-changes.md

```diff
@@ -21,26 +21,27 @@ exercises: 0
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
 First let's make sure we're still in the right directory.
-You should be in the `planets` directory.
+You should be in the `recipes` directory.
 
 ```bash
-$ cd ~/Desktop/planets
+$ cd ~/Desktop/recipes
 ```
 
-Let's create a file called `mars.txt` that contains some notes
-about the Red Planet's suitability as a base.
+Let's create a file called `guacamole.md` that contains the basic structure to
+have a recipe.
 We'll use `nano` to edit the file;
 you can use whatever editor you like.
 In particular, this does not have to be the `core.editor` you set globally earlier. But remember, the bash command to create or edit a new file will depend on the editor you choose (it might not be `nano`). For a refresher on text editors, check out ["Which Editor?"](https://swcarpentry.github.io/shell-novice/03-create.html#which-editor) in [The Unix Shell](https://swcarpentry.github.io/shell-novice/) lesson.
 
 ```bash
-$ nano mars.txt
+$ nano guacamole.md
 ```
 
-Type the text below into the `mars.txt` file:
+Type the text below into the `guacamole.md` file:
 
 ```output
-Cold and dry, but everything is my favorite color
+# Ingredients
+# Instructions
```

Comment:
```suggestion
## Ingredients
## Instructions
```

For better accessibility, files should contain only a single top-level (`#`) heading.

Reply:
This suggestion superseded by @dpshelio's superior version in #1013